lib-frontend-estree/src/func/post_parse.rs

```diff
@@ -2029,13 +2046,15 @@ fn post_parse_call_func_with_params_helper(
     num_locals: usize, // current number of IR locals
     filename: Option<&str>,
     ir_program: &mut ir::Program,
+    is_tail: &bool,
 ) -> Result<ir::Expr, CompileMessage<ParseProgramError>> {
     let args: Box<[ir::Expr]> = args_iter
-        .map(|arg| post_parse_expr(arg, parse_ctx, depth, num_locals, filename, ir_program))
+        .map(|arg| post_parse_expr(arg, parse_ctx, depth, num_locals, filename, ir_program, &false))
         .collect::<Result<Box<[ir::Expr]>, CompileMessage<ParseProgramError>>>()?;
     Ok(ir::Expr {
         vartype: Some(ir::VarType::Any),
```

Comment:
Shouldn't a tail call Expr have a vartype of `None` (to indicate that it never returns) ?